src/dev/i18n/extractors/react.js

```diff
@@ -29,9 +29,11 @@ import {
   extractValuesKeysFromNode,
   checkValuesProperty,
 } from '../utils';
-import { DEFAULT_MESSAGE_KEY, CONTEXT_KEY, VALUES_KEY } from '../constants';
+import { DEFAULT_MESSAGE_KEY, VALUES_KEY } from '../constants';
 import { createFailError } from '../../run';
 
+const CONTEXT_KEY = 'description';
```

Comment:
please move it back to constants and make it universal for angular, react and node.js